File: Oke_server/oke/src/main/java/com/bnuz/oke/entity/VoteStudent.java
package com.bnuz.oke.entity;

import java.io.Serializable;

/**
 * VoteStudent
 *
 * @author handsome
 * @date 2020年 06月02日 13:59:19
 */
public class VoteStudent implements Serializable {

	private static final long serialVersionUID = 697839377051194910L;

	private Student student;

	private Vote vote;

	private VoteChoice voteChoice;

	public Student getStudent() {
		return student;
	}

	public void setStudent(Student student) {
		this.student = student;
	}

	public Vote getVote() {
		return vote;
	}

	public void setVote(Vote vote) {
		this.vote = vote;
	}

	public VoteChoice getVoteChoice() {
		return voteChoice;
	}

	public void setVoteChoice(VoteChoice voteChoice) {
		this.voteChoice = voteChoice;
	}

	@Override
	public String toString() {
		return "VoteStudent{" +
				"student=" + student +
				", vote=" + vote +
				", voteChoice=" + voteChoice +
				'}';
	}
}

File: Oke_client/Oke_teacher/Oke_teacher/Entity/VoteStudent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Entity
{
    /// <summary>
    /// 学生投票信息封装类
    /// </summary>
    [Serializable]
    public class VoteStudent
    {
        //学生
        public Student student { set; get; }
        //投票
        public Vote vote { set; get; }
        //学生的选择
        public VoteChoice voteChoice { set; get; }
    }
}
